platform/web/detect.py

```diff
@@ -199,6 +199,11 @@ def configure(env: "SConsEnvironment"):
     cc_version = get_compiler_version(env)
     cc_semver = (cc_version["major"], cc_version["minor"], cc_version["patch"])
 
+    # Minimum emscripten requirements.
+    if cc_semver < (3, 1, 62):
+        print_error("GDExtension support requires emscripten >= 3.1.62, detected: %s.%s.%s" % cc_semver)
```

Comment:
The error talks about GDExtension but you made it a requirement for all Web builds.